src/lib.rs

```diff
@@ -54,6 +54,8 @@ extern crate failure;
 #[cfg(test)]
 extern crate test;
 
+extern crate coarsetime;
+
```

Comment:
Why do we need it?

Reply:
> Why do we need it?

`make_auto_flush_static_metric!` uses coarsetime

Reply:
In Rust 2018, `extern crate` should be unnecessary in most cases.